code/index.html

```diff
@@ -1,30 +1,181 @@
-  </head>
-  <body>
-    <!-- video or image as a header is cool :) -->
-
-    <!-- Signup form -->
-
-    <section class="accordion">
-      <!-- Accordion -->
-      <h1>FAQ</h1>
-
-      <div class="question" id="section1">Section 1</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
+
+<head>
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Project 3</title>
+  <link rel="stylesheet" href="./style.css">
+</head>
+
+<body>
+  <header> 
+    <h1>The Friday Bouquet</h1>
+
+    <video autoplay muted loop>
+      <source src="./videos/flowers.mp4" type="video/mp4">
+      <p>This browser doesn't support video.</p>
+    </video> 
+
+    <picture class="hero-image">
+      <source srcset="./images/mobilBukett.jpg" media="(max-width: 667px)"/>
+      <source srcset="./images/bukett.jpg" media="(max-width: 1024px)"/> 
+      <img src="./images/flowersStill.jpg" alt="image of flowers"/> 
+    </picture>
+
+  </header>
+
+  <section class="intro-text">
+    <p>Get fresh flowers every Friday - Delivered to your door</p>
+  </section>
+
+  <section class="box-container">
+  
+    <section class="form-container">
+      <form class="form" 
+        action="http://httpbin.org/anything" 
+        target="_blank" 
+        method="POST"
+      >
+      <h2>Subscribe</h2>
+        <div class="form-group1"></div>
+          <div class="form-part">
+            <label for="name">Name</label>
+            <input 
+              type="text" 
+              name="nameValue"
+              id="name" 
+              placeholder="Namn Namnsson" 
+              required 
+            />
+          </div>
+
+          <div class="form-part">
+            <label for="email">Email</label>
+            <input 
+              type="text" 
+              name="emailValue"
+              id="email" 
+              placeholder="email@email.com" 
+              required 
+            />
+          </div>
+          
+          <div class="form-part">
+            <label for="password">Password</label>
+            <input 
+              type="text" 
+              name="passwordValue"
+              id="password"
+              required 
+            />
+          </div>
+          
+          <div class="form-checkbox">
+            <label for="termsCon">
+            Accept Terms & Conditions 
+            <input 
+              type="checkbox" 
+              name="termsConditions"
+              id="termsCon" 
+              required 
+            />
+            <span class="checkmark"></span>
+          </label>
+          </div>  
+        </div>
+
+        <p class="form-text">Size of bouquet</p> 
+
+        <div class="form-group2">
+          <label class="radiobutton-container">
+          Small
+          <input 
+            type="radio" 
+            name="size"
+            value="small" 
+            checked
+          />
+          <span class="custom-radio"></span>
```

Comment:
Hi Maksymilian! Thank you for your feedback. Could you please show me the correct way to indent in this case? so I can learn and do it the correct way for this week's project. Thank you in advance!

Reply:
Hey Ellen, sure! I see it's hard to see it over here because comments on Github only shows 3 lines above the one with error. Please take a look over here https://github.com/ellenbjo/project-business-site/blob/master/code/index.html - it's your file `index.html` but without comments. As you can see in line 92 you start to write code nested inside `label` element, but it has same indent as `label` itself (and should have two spaces more). Hope that will help :)